test/onnx/test_pytorch_onnx_caffe2_quantized.py

```diff
@@ -144,7 +157,7 @@ def forward(self, x):
                 return self.dequant(res)
 
         x = np.random.rand(1, 2, 3, 4).astype("float32")
-        self.generic_test(QUpsampleModule(), (x,), input_names=["x"])
+        self.generic_test(QUpsampleModule(), (x,), input_names=["x"], decimal=0)
```

Comment:
Suggest using decimal=5 for cases where we expect an exact match.

Reply:
Oops, yeah my bad